src/cmdstan-guide/pathfinder_intro.Rmd

```diff
@@ -0,0 +1,129 @@
+# Variational Inference using Pathfinder
+
+The CmdStan method `pathfinder` uses the Pathfinder algorithm of @zhang_pathfinder:2022.
+Pathfinder is a variational method for approximately
+sampling from differentiable log densities. Starting from a random
+initialization, Pathfinder locates normal approximations to the target
+density along a quasi-Newton optimization path, with local covariance
+estimated using the inverse Hessian estimates produced by the
+L-BFGS optimizer.  Pathfinder returns draws from the Gaussian approximation
+with the lowest estimated Kullback-Leibler (KL) divergence to the true
+posterior.
+
+Pathfinder differs from the ADVI method in that it uses quasi-Newton
+optimization on the log posterior instead of stochastic gradient descent (SGD).
```

Comment:
"stochastic gradient descent (SGD)" should continue with: 

on the Monte Carlo computation of the evidence lower bound (ELBO).